Doc/glossary.rst

```diff
@@ -221,6 +221,14 @@ Glossary
       statement by defining :meth:`__enter__` and :meth:`__exit__` methods.
       See :pep:`343`.
 
+    context variable
+      A variable which can have different values depending on it's context.
+      This is similar to TLS(Thread Local Storage) in which each execution
+      thread may have a different value for a variable. In case of context
+      variable, there may be sevaral contexts in one execution thread and it is
+      used to keep track of variables in concurrent asynchronous tasks.
+      See `https://docs.python.org/3/library/contextvars.html`.
```

Comment:
you should use `:mod:` rest role instead of a direct http link. look at other places.

Reply:
yes. done.

Reply:
As i am new to python org. I am learning these things now.

Reply:
You're doing great! :) Sorry for the delayed review.